apps/src/userHeaderEventLogger/userHeaderEventLogger.js

```diff
@@ -198,9 +198,11 @@ $(document).ready(function () {
   const headerCreateMenu = document.getElementById('header_create_menu');
   const pageUrl = window.location.href;
 
-  if (getScriptData('isSignedOut')) {
-    addSignedOutMetrics(pageUrl, headerCreateMenu);
-  } else {
-    addSignedInMetrics(pageUrl, headerCreateMenu);
+  if (!pageUrl.includes('/global/')) {
```

Comment:
out of curiousity, why wouldn't we want to log for this URL?

Reply:
When I first started I changed the header and because of those header changes I think it was failing.... Just figured it was simpler to start with it not logging and could be updated. I'll take another look now to see if it makes sense to keep it in

Reply:
As long as no one is relying on that data, I think it's fine.